### ISSUE TITLE
Maximum call stack size exceeded: flex-flow

### ISSUE THREAD

**Issue by vote539** [krkn/kouto-swiss#56]
Using this simple Stylus file,

```stylus
@import "kouto-swiss"

body {
	display: flex
	flex-flow: row wrap
}
```

When Grunt compiles it, I get this error.

```
>> File "app/styl/hello.styl" changed.
Running "stylus:dev" (stylus) task
>> RangeError: app/styl/hello.styl:5:17
>>    1| @import "kouto-swiss"
>>    2| 
>>    3| body {
>>    4| 	display: flex
>>    5| 	flex-flow: row wrap
>> ----------------------^
>>    6| }
>>    7| 
>> 
>> Maximum call stack size exceeded
>>     at flex-flow() (/path/to/node_modules/kouto-swiss/lib/kouto-swiss/mixins/vendors-flex.styl:78:35)
>>     at "body " (app/styl/hello.styl:2:22)
```

**Comment by leny** [comment]
Hi, thanks for the report.

I will take a look right now. :)

**Comment by leny** [comment]
Fixed & published as version `0.11.13`.

Thanks again for the report.

**Comment by vote539** [comment]
Great, thanks for the fast response :+1:
